src/components/SocialListItem.tsx

```diff
@@ -1,42 +1,64 @@
 // Libraries
-import React from "react"
-import styled from "@emotion/styled"
+import React, { useMemo } from "react"
+import { Flex, Box, chakra, Icon } from "@chakra-ui/react"
+import {
+  FaTwitter,
+  FaYoutube,
+  FaDiscord,
+  FaRedditAlien,
+  FaStackExchange,
+  FaGlobe,
+} from "react-icons/fa"
 
 // Components
-import Icon from "./Icon"
+import { socialColors } from "./Icon"
 
 // Styles
-const Item = styled.div`
-  display: flex;
-  width: 100%;
-  padding: 0.5rem 0;
-  align-items: center;
-`
-
-const StyledIcon = styled(Icon)`
-  padding-right: 0.75rem;
-`
-
-const ChildrenContainer = styled.div`
-  font-style: italic;
-
-  > a {
-    font-style: normal;
-  }
-`
+const StyledIcon = chakra(Icon, {
+  pr: 3,
+  boxSize: "2.5rem",
+})
```

Comment:
As mentioned before, we are not using our custom `Icon` component anymore. You can check [the migration guide](https://github.com/ethereum/ethereum-org-website/blob/dev/docs/chakra-migration-guide.md) and see how to use Icons from now on. Basically, we want to use the native Chakra Icon component.

Reply:
Resolved